### PR TITLE
Weather

### PR DESCRIPTION
**Pull request by mariuszstr**
-Weather functions and classes.
-Tests.

### CODE REVIEW

File: engine/db/weather.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    HUGE_SNOW = 9
+
+
+class Weather:
+    def __init__(self, weather_type, temperature):
+        self.weather_type = weather_type
+        self.temperature = temperature
+
+
+def generate_weather_type(temperature):
```
Comment (RCAnthrax) [inline engine/db/weather.py:23]: missing 4 spaces?

File: engine/heat/calculate_heat_time.py [hunk 2]
```diff
@@ -0,0 +1,2 @@
+def calculate_heat_time():
+    pass
```
Comment (RCAnthrax) [inline engine/heat/calculate_heat_time.py]: No newline at the end

**Review by RCAnthrax — CHANGES_REQUESTED** [review]

File: engine/db/weather.py (hunk 1 above)
Comment (mariuszstr) [reply engine/db/weather.py:23]: It's function, not method. All is ok.

File: engine/heat/calculate_heat_time.py (hunk 2 above)
Comment (mariuszstr) [reply engine/heat/calculate_heat_time.py]: Done.

**Pull request merged**